### PR TITLE
Comment and clean

### PR DESCRIPTION
**Pull request by sisyphusSmiling**
Closes: #108

### CODE REVIEW

File: contracts/HybridCustody.cdc [hunk 1]
```diff
@@ -6,27 +6,32 @@ import "CapabilityFactory"
...
-    would then also be able to manage the child account's links as it sees fit.
-
-Contributors (please add to this list if you contribute!):
-- Austin Kline - https://twitter.com/austin_flowty
-- Deniz Edincik - https://twitter.com/bluesign
-- Giovanni Sanchez - https://twitter.com/gio_incognito
-- Ashley Daffin - https://twitter.com/web3ashlee
-*/
+/// HybridCustody defines a framework for sharing accounts via account linking. In the contract, you will find three
+/// main resources being used:
```
Comment (joshuahannan) [inline contracts/HybridCustody.cdc:10]:
```suggestion
/// HybridCustody defines a framework for sharing accounts via account linking.
/// In the contract, there are three main resources:
```

File: contracts/HybridCustody.cdc [hunk 2]
```diff
@@ -6,27 +6,32 @@ import "CapabilityFactory"
...
+///
+/// 1. OwnedAccount - A resource which maintains an AuthAccount Capability, and handles publishing and revoking access
+///    of that account via another resource called a ChildAccount
+/// 2. ChildAccount - A second resource which exists on the same accounta as the OwnedAccount. The ChildAccount
+///    resource is the capability which is shared to a parent account. Each child has its own set of rules designating
+///    what Capability types can be retrieved, and what underlying type the capability points to can be given.
```
Comment (joshuahannan) [inline contracts/HybridCustody.cdc:16]: This sentence is a little confusing

File: contracts/HybridCustody.cdc [hunk 3]
```diff
@@ -53,47 +70,46 @@ pub contract HybridCustody {
...
     pub resource interface OwnedAccountPublic {
-        pub fun getParentsAddresses(): [Address]
+        /// Returns the addresses of all parent accounts
+        pub fun getParentAddresses(): [Address]
 
-        // getParentStatuses
-        // Returns all parent addresses of this child account, and whether they have already
-        // redeemed the parent has redeemed the account (true) or has not (false)
+        /// Returns all parent addresses of this child account, and whether they have already redeemed the parent has
+        /// redeemed the account (true) or has not (false)
```
Comment (joshuahannan) [inline contracts/HybridCustody.cdc:90]: wording is kind of weird here

**Review by joshuahannan — APPROVED** [review]

File: contracts/HybridCustody.cdc [hunk 4]
```diff
@@ -226,51 +228,52 @@ pub contract HybridCustody {
...
         pub fun setManagerCapabilityFilter(cap: Capability<&{CapabilityFilter.Filter}>?, childAddress: Address) {
             pre {
                 cap == nil || cap!.check(): "Invalid Manager Capability Filter"
             }
         }
     }
 
-    // Functions anyone can call on a manager to get information about an account such as
-    // What child accounts it has
+    /// Functions anyone can call on a manager to get information about an account such as What child accounts it has
```
Comment (austinkline) [inline contracts/HybridCustody.cdc:247]:
```suggestion
    /// Functions anyone can call on a manager to get information about an account such as what child accounts it has
```

**Review by austinkline — APPROVED** [review]

**Pull request merged**